File: Aula03_23-08-23/busca_sequencial.mjs
function buscaSequencial(vetor, valorBusca){
    for(let x = 0; x < vetor.length; x++){
        // == é igual (não verifica tipo)?, === é extremamente igual (verifica tipo)?    
        if (valorBusca === vetor[x]) return x
    }
    return -1
}

import { nomes } from './data/vetor-nomes.mjs'

console.time("Buscando...")
console.log('Posição MATHEUS: ' + buscaSequencial(nomes, "ALEXANDRE"))
console.timeEnd("Buscando...")  // precisa ter a mesma 'descrição' do comando 'console.time()'


// const frutas = ['maçã', 'uva', 'graviola', 'mamao']

// var valor = require("Entre com uma fruta: ")

// console.log(buscaSequencial(frutas, valor))
// // console.log(buscaSequencial(frutas, "mamao"))

File: Aulas/Aula03_23-08-23/busca_sequencial.py
vetor1 = [1, 2, 3, 4, 5, 6, 7, 8]

valor = int(input("Entre com um valor: "))
for n in vetor1:
    if valor == vetor1[n]:
        posicao = vetor1[n]
        break
    else:
        pass

print(f"O valor {valor} está na posição: {posicao}")

        
